ci/build.yml

```diff
@@ -0,0 +1,5 @@
+steps:
+  - script: |
+        cargo rustc -- -D warnings
+    workingDirectory: '$(Build.SourcesDirectory)/${{parameters.crate_path}}'
+    displayName: Run cargo rustc -- -D warnings on ${{parameters.crate_path}}
```

Comment:
I would just call this "Check for build warnings" (this is the name that shows up in the report on the web dashboard).

Reply:
I was trying to be consistent with the apparent naming convention of the other scripts, which seemed to be more verbose. Should I change them as well? See e.g. rustfmt and test.

Reply:
Consistency is good, happy with this as is!